packages/studioCMS/src/integrations/studioCMSDashboard/routes/authroutes/login/index.astro

```diff
@@ -82,26 +79,27 @@ if (locals.isLoggedIn) {
           </div>
         </div> 
       )}
-      <!-- End of Username and Password Provider-->
 
-      <sl-alert id="form-error" variant="danger" duration="4000" closable>
-        <sl-icon slot="icon" name="exclamation-octagon"></sl-icon>
+      <Alert id="form-error" variant="danger" duration={4000} closable>
+        <Icon slot="icon" name="exclamation-octagon" />
         <strong>Something went wrong!</strong><br />
         <div id = "form-error-message"></div>
```

Comment:
do attributes work with spaces like this? `id = "foo"` ????

Reply:
They do, ironically enough... nice catch btw